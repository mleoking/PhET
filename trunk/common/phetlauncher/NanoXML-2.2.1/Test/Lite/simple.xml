<?xml version="1.0" encoding="US-ASCII"?>
<FOO>
    <BAR/>
    <BAR x='1'/>
    <BAR x='1' y='2'/>
    <BAR>
        <BAR/>
        <BARBAR/>
        <BA/>
        <FOO/>
    </BAR>
    <BAR>blah</BAR>
    <BAR x='1'>blah</BAR>
    <BAR><![CDATA[<&>'"]]></BAR>
    <BAR>a<![CDATA[b]]>c</BAR>
    <BAR>&lt;&amp;&gt;&quot;&apos;&#33;&#x3A;&#x2030;</BAR>
</FOO>

