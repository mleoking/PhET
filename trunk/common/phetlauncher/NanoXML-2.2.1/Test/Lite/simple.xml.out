<FOO><BAR/><BAR x="1"/><BAR x="1" y="2"/><BAR><BAR/><BARBAR/><BA/><FOO/></BAR><BAR>blah</BAR><BAR x="1">blah</BAR><BAR>&lt;&amp;&gt;&apos;&quot;</BAR><BAR>abc</BAR><BAR>&lt;&amp;&gt;&quot;&apos;!:&#x2030;</BAR></FOO>
